libs/sdk-core/src/models.rs

```diff
@@ -1182,8 +1182,11 @@ pub enum LnUrlCallbackStatus {
 /// Represents a LNURL-pay request.
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct LnUrlPayRequest {
+    /// The [LnUrlPayRequestData] returned by [BreezServices::parse_input]
     pub req_data: LnUrlPayRequestData,
+    /// The amount in millisatoshis for this payment
     pub amount_msat: u64,
+    /// An option comment for this payment
```

Comment:
NIT: option is more of a rust specific jargon afaik, better to use the word optional here.